azkaban-web-server/src/web/js/azkaban/model/svg-graph.js

```diff
@@ -27,6 +27,24 @@ azkaban.GraphModel = Backbone.Model.extend({
   addFlow: function (data) {
     this.processFlowData(data);
     this.set({'data': data});
+    this.set({'nodeIdsAndNestedIds': this.getAllNodeIdsAndNestedIds(data)});
+  },
+
+  getAllNodeIdsAndNestedIds: function (rootNode) {
```

Comment:
To ensure all clients are updated with these UI changes, for all modified .js files, please add `?v=<timestamp>` in all places where they are imported/referenced. For example:

`<script type="text/javascript" src="${context}/js/azkaban/view/flow-execute-dialog.js?v=1569610022"></script>`

Reply:
Added commit